server/v2/telemetry/metrics.go

```diff
@@ -0,0 +1,187 @@
+package telemetry
```

Comment:
moved this here to avoid the sdk import. Also I believe it makes sense for this to live where its instantiated in the new system instead of separately. 

For the wrapper.go file, it could get moved to core or something high level to avoid sdk import in modules

Reply:
Yes, having this in server makes sense.